bentoml/deployment/utils.py

```diff
@@ -44,3 +52,77 @@ def process_docker_api_line(payload):
                     )
                 elif "stream" in line_payload:
                     logger.info(line_payload['stream'])
+
+
+INSTALL_TARGZ_TEMPLATE = """\
+#!/bin/bash
+
+for filename in ./bundled_dependencies/*.tar.gz; do
+    [ -e "$filename" ] || continue
+    pip install "$filename" --ignore-installed
+done
+"""
+
+
+def add_local_bentoml_package_to_repo(archive_path):
+    try:
+        module_location, = importlib.util.find_spec('bentoml').submodule_search_locations
+    except AttributeError:
+        # python 2.7 doesn't have importlib.util, will fall back to imp instead
+        import imp
+        _, module_location, _ = imp.find_module('bentoml')
+    bentoml_setup_py = os.path.abspath(os.path.join(module_location, '..', 'setup.py'))
+
+    if not os.path.isfile(bentoml_setup_py):
+        raise KeyError('"setup.py" for Bentoml module not found')
```

Comment:
keyError not suitable for this case,  use `assert os.path.isfile(bentoml_setup_py), '"setup.py" for Bentoml module not found'`